lohica-js-network/src/reducers/inputValidate.js

```diff
@@ -0,0 +1,200 @@
+const initState = {
+    regForm: {
+        name: '',
+        surname: '',
+        midName: '',
+        email: '',
+        age: '', 
+        gender:'',
+        ageValid:'',
+        imageStatus: null,
+        nameValid: false,
+        nameValidMessage: null,
+        nameValidMessageShow: false,
+        surnameValid: false,
+        surnameValidMessage: null,
+        surnameValidMessageShow: false,
+        midNameValid: false,
+        midNameValidMessage: null,
+        midNameValidMessageShow: false,
+        emailValid: false,
+        emailValidMessage: null,
+        emailValidMessageShow: false,
+        genderValid: null,
+        genderValidStat: false,
+        imageData: '',
+        imageValid: false,
+        imageValidShow: false,
+        regToLoginRedirect: {
+            password: '',
+            message: '',
+            show: false,
+            alertStyle: ''
+        },
+        regValidateState: {
+            alertStyle: null,
+            showWarning: false,
+            show: false,
+            password: '',
+            message: '',
+        },
+        successReg: false
+    },
+    loginForm:{
+        email: '',
+        password: '',
+        emailValid: false,
+        emailValidMessage: null,
+        emailValidMessageShow: false
+    }
+};
+
+const inputValidate = (state=initState, action) => {
+    let ap = action.payload;
+    switch(action.type){
+        case 'VALIDATE_NAME':
```

Comment:
action type copy pasted should a kind of constant ACTION_TYPES.VALIDATE_NAME

Reply:
done